data/stoa0054/stoa006/stoa0054.stoa006.perseus-eng1.xml

```diff
@@ -54,8 +61,9 @@
 
     <profileDesc>
       <langUsage>
-	<language id="la">Latin</language>
-	<language id="greek">Greek</language>
+   <language id="eng">English</language>
```

Comment:
@AlisonBabeu 
We don't need the language id to include English if it's only for text that appears in the header.
I'm not sure if that's why this is here, but this would apply to every file if that were the case and it's not necessary.
I did notice that here and elsewhere, we didn't always indicate the header is in English, so I need to globally fix that, especially in OGL.